src/outputs/restart.hpp

```diff
@@ -116,6 +116,7 @@ class RestartReader {
   std::vector<T> ReadDataset(const char *name, size_t *count = nullptr) {
     // Returns entire 1D array.
     // status, never checked.  We should...
+    std::vector<T> data;
```

Comment:
Is this `ifdef` behavior actually what we want? (and/or is this a leftover from Athena++?)
I'm asking as having the code fail seems like a preferable solution rather than returning an empty object (due to lack of implementation or similar).